improver/calibration/ensemble_calibration.py

```diff
@@ -966,30 +1029,26 @@ def _spatial_domain_match(self):
                 self.current_forecast.coord(axis=axis).points[0],
                 self.current_forecast.coord(axis=axis).points[-1],
             ]
-            if not np.allclose(
-                current_forecast_points, self.coefficients_cube.coord(axis=axis).bounds
-            ):
-                raise ValueError(
-                    msg.format(
-                        axis,
-                        current_forecast_points,
-                        self.coefficients_cube.coord(axis=axis).bounds,
+            for coeff_cube in self.coefficients_cubelist:
+                if not np.allclose(
+                    current_forecast_points, coeff_cube.coord(axis=axis).bounds
+                ):
```

Comment:
I've modified this, following the suggestion above to use the `coord.collapsed` coordinates for simplicity. However, this does not include the checking of the coordinate system on the coordinates because, it seems like for scalar cubes, the coordinate system is not stored on the coordinate when saved. This can be seen by checking the output of the `estimate-emos-coefficients` CLI, prior to saving, where the x and y coordinates do have a coord system, however, no coord system is present on the x and y coordinates when the resulting file is loaded from disk. The coordinate system is also not present on the cube itself, so I haven’t been able to check that either.